src/nested_pandas/nestedframe/core.py

```diff
@@ -13,7 +13,26 @@
 from nested_pandas.series import packer
 from nested_pandas.series.dtype import NestedDtype
 
-from .utils import _ensure_spacing
+from ..series.packer import pack_sorted_df_into_struct
+from .utils import NestingType, check_expr_nesting
+
+
+class NestedSeries(pd.Series):
```

Comment:
Is it actually "FlatSeries"? Or maybe `_FlatSeries` because it is not part of the public interface?

Reply:
Good question.  While technically it does cache the flattened nest, it does retain a reference to the original nest name and its nest siblings, so it knows that it is part of a nest.  I do think it should be public and not private because it does sometimes become the return value of `NestedFrame.eval`, depending on the expression.

Reply:
Makes sense.  I'll make it private and give it a name that doesn't suggest generality.  Maybe `_SeriesFromNest`?

Reply:
Sounds good, thank you!